core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/Resource.scala

```diff
@@ -86,6 +87,29 @@ object Resource extends ConfigFormat {
     }
   }
 
+  final class DMGFile(val path: Path) extends IntellijResource {
+    def installTo(target: Path): Unit = {
+      import sys.process._
+
+      // create tmp directory where disk image will be attached
+      val dmgDirPath = Paths.get(System.getProperty("java.io.tmpdir")).resolve("dmg_dir")
+      dmgDirPath.createDirectory()
+      val dmgDir = dmgDirPath.toString
+
+      try {
+        // attach disk image from .dmg file to local filesystem
+        s"hdiutil attach -mountpoint $dmgDir ${path.toString}".!
```

Comment:
for consistency, it would be better to use probe's shell api.
Shell.run("hdiutil", "attach", "-mountpoint", dmgDir.toString, path.toString)